compose/material3/material3/src/androidMain/res/values-en-rGB/strings.xml

```diff
@@ -20,6 +20,7 @@
     <string name="m3c_dialog" msgid="7617233117134790350">"Dialogue"</string>
     <string name="m3c_dropdown_menu_expanded" msgid="2360841780724299882">"Expanded"</string>
     <string name="m3c_dropdown_menu_collapsed" msgid="3177828188723359358">"Collapsed"</string>
+    <string name="m3c_dropdown_menu_toggle" msgid="8687821690726149911">"Toggle drop-down menu"</string>
```

Comment:
TODO: We need to re-gen material3 multiplatform translations

Reply:
https://youtrack.jetbrains.com/issue/CMP-1595/Regenerate-material3-translations-for-1.7.0

Reply:
@igordmn already done in #1418